README.md

```diff
@@ -20,15 +20,33 @@
 
 # Prefect
 
-Prefect is an orchestrator for data-intensive workflows. It's the simplest way to transform any Python function into a unit of work that can be observed and orchestrated. With Prefect, you can build resilient, dynamic workflows that react to the world around them and recover from unexpected changes. With just a few decorators, Prefect supercharges your code with features like automatic retries, distributed execution, scheduling, caching, and much more. Every activity is tracked and can be monitored with the Prefect server or Prefect Cloud dashboard.
+Prefect is a workflow application framework. It's the simplest way to deploy Python code as an interactive workflow application that can be observed and orchestrated.
```

Comment:
I would not go so hard on "workflow application framework".
```suggestion
Prefect is a developer platform for building, observing, and triaging workflows. Powered by a Python library, it pairs with a responsive dashboard and observability for getting a complete picture of your workflows.
```

Reply:
While I'm comfortable emphasizing the pythonic nature of the SDK, I don't think "Powered by a Python library" is particularly accurate or informative here.